doc/de/LIESMICH

```diff
@@ -3137,11 +3136,11 @@ http://wiki.scummvm.org/index.php/Compiling_ScummVM/Mac_OS_X_Crosscompiling
 Eine ausführliche Liste derjenigen, die ScummVM ermöglicht haben, finden Sie
 unter:
 
-  http://www.scummvm.org/credits
+  https://www.scummvm.org/credits
```

Comment:
Technically not necessary but for the sake of consistency I'd add the trailing slash here as well.